### PR TITLE
feat: User stats

### PR DESCRIPTION
**Pull request by Shivani-1524**
added user performance stats and profile page. this is a private route.

credentials: 
email - test@email.com
pswd - test

Files to review:
ProfilePage folder
quiz-reducer.js
quizstats-requests.js

### CODE REVIEW

File: src/Pages/ProfilePage/ProfilePage.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react'
+import { handleGetStat } from '../../Utils/quizstats-requests'
+import { Navbar } from '../../Components/Navbar/Navbar'
```
Comment (gauravsinhaweb) [inline src/Pages/ProfilePage/ProfilePage.js:3]: Suggestion: folder name should be in lowercase.

File: src/Pages/QuizPages/QuizPage.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+const QuizPage = () => {
+
+    const { quizState, quizDispatch } = useQuiz()
+    const [quizSubmit, setQuizSubmit] = useState(false)
+    const [showModal, setShowModal] = useState(false)
+    const { quizId } = useParams()
+    const [questionsData, setQuestionsData] = useState(false)
+    const { questions, quizAnswer } = questionsData
+    const quizHeading = questionsData?.quizName?.split(':')[1].toUpperCase()
```
Comment (gauravsinhaweb) [inline src/Pages/QuizPages/QuizPage.js:20]: missing semicolon ig

**Review by gauravsinhaweb — APPROVED** [review]
couldn't find any major improvement, well coded, and liked the idea of earning coins in user stats. you're good to merge!

File: src/Pages/QuizPages/QuizPage.js (hunk 2 above)
Comment (Shivani-1524) [reply src/Pages/QuizPages/QuizPage.js:20]: yep inconsistent with semicolons but will fix it later

**Pull request merged**